components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -2413,6 +2413,7 @@ document.observe('xwiki:dom:loaded', function()
         this._builder.rejectedGeneIDsInput.value = '';
         for (var gene in this._builder.genes) {
           (this._builder.genes[gene].status == 'rejected') &amp;&amp; (_this._builder.rejectedGeneIDsInput.value += gene + ' ');
+          (this._builder.genes[gene].status == 'rejected_candidate') &amp;&amp; (_this._builder.rejectedGeneIDsInput.value += gene + ' ');
```

Comment:
Shouldn't both values be considered here?

Reply:
It seems that line 2415 is checking 'rejected' status, or I still did not get the question

Reply:
You are right, I thought line 2415 was removed and line 2416 added in its place.